### PR TITLE
WFP-1889: Updated organisation overview tables to use MoJ sortable table

### PR DESCRIPTION
**Pull request by sp-watson**

### CODE REVIEW

File: app/assets/sass/application.scss [hunk 1]
```diff
@@ -203,6 +211,16 @@ table.override-gov-uk-table tr{
   line-height: 40px;
 }
 
+.table-first-visible-child {
```
Comment (sp-watson) [inline app/assets/sass/application.scss:214]: There is CSS to select the first visible child, but this is overriding an MoJ sortable table style which is for the first child. I chose to make it more explicit.

File: app/views/includes/organisation-overview.njk [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
+                <th headers="cases" class="govuk-table__header govuk-table__header--numeric" aria-sort="none" id="total">Total</th>
             </tr>
         </thead>
         <tbody>
-            {% for member in overviewDetails.rows %}
-            <tr>
-              <td headers="name om"><a class="govuk-link" href="/{{ workloadType }}/{{ childOrganisationLevel }}/{{ member.linkId }}">{{ member.name }}</td>
+          {% for member in overviewDetails.rows %}
+          <tr class="govuk-table__row">
+            {## Hidden rows to allow for moj sortable table to identify columns correctly (thanks to having 2 header rows) ##}
```
Comment (sp-watson) [inline app/views/includes/organisation-overview.njk:34]: Sigh

**Review by AndySweilam — DISMISSED** [review]
⭐

**Review by AndySweilam — APPROVED** [review]
💯

**Pull request merged**